tsconfig.json

```diff
@@ -6,19 +6,13 @@
     "module": "CommonJS",
     "moduleResolution": "node",
     "outDir": "./dist",
-    "isolatedModules": true,
+    "isolatedModules": false,
```

Comment:
possibly contentious change here.  The tests fail without this on line 49 of `cli.ts` because `require('..').main` returns `undefined`.  I don't know enough about this option to offer more info or alternatives.

Reply:
updated tests to no longer require this